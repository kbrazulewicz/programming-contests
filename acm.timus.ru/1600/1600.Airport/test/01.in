4 15.0
15.0 1.4 -3.0 3.0 54.0 130.0
65.0 23.0 4.0 5.0 3.0 34.4
23.0 57.0 5.0 8.0 4.0 31.4
7.0 34.0 7.0 123.0 76.0 54.4
