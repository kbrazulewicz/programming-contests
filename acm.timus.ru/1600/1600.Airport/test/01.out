ALARM!
0.124 3 4
